### PR TITLE
Dev Console- Adding Endpoints

### PR DESCRIPTION
**Pull request by saumya1singh**
PR for #14195

### CODE REVIEW

File: extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/resources/dev-templates/endpoints.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  </div>
+  <div class="col">
+   {endpoint.produces}
+  </div>
+ </div>
+ {/if}
+ {/for}
+</header>
+
+<header>
```
Comment (FroMage) [inline extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/resources/dev-templates/endpoints.html:26]: Is the empty section intentional?

**Review by FroMage — CHANGES_REQUESTED** [review]

File: extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/resources/dev-templates/endpoints.html (hunk 1 above)
Comment (SaumyaSingh1) [reply extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/resources/dev-templates/endpoints.html:26]: Yes, I was thinking to add `Additional endpoints` category here.

File: extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/resources/dev-templates/endpoints.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  <div class="col-1">
+   Produces
+  </div>
+  <div class="col">
+   {endpoint.produces}
+  </div>
+ </div>
+ {/if}
+ {/for}
+</header>
```
[the remark this answers is not in the document]
Comment (SaumyaSingh1) [reply extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/resources/dev-templates/endpoints.html:24]: > Unless you have some custom CSS specific to `header`, I think you can get rid of it. I'm not completely sure, but the way https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header describes those elements doesn't look like what you're using them for. Unless you have a reason?

I though of adding separate headers for each category like `REST Resources`, `additional endpoints` ..

[the remark this answers is not in the document]
Comment (FroMage) [reply extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/resources/dev-templates/endpoints.html:24]: OK, fine, let's start with this.

File: extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/resources/dev-templates/endpoints.html (hunk 1 above)
Comment (FroMage) [reply extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/resources/dev-templates/endpoints.html:26]: OK

**Comment by FroMage** [comment]
The CI failure is unrelated:

```
2021-01-15T06:36:44.3821109Z [INFO] Running io.quarkus.reactive.datasource.runtime.ConnectionPoolsClosedTest
2021-01-15T06:36:44.4334797Z [ERROR] Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.067 s <<< FAILURE! - in io.quarkus.reactive.datasource.runtime.ConnectionPoolsClosedTest
2021-01-15T06:36:44.4339579Z [ERROR] io.quarkus.reactive.datasource.runtime.ConnectionPoolsClosedTest.connectionsFromOtherThreadsGetClosed  Time elapsed: 0.034 s  <<< FAILURE!
2021-01-15T06:36:44.4342435Z java.lang.AssertionError: COM01003: Internal error: Assertion failure: Expected boolean value to be false
```

**Review by FroMage — APPROVED** [review]
OK this is a great starting point, let's get this in!

**Pull request merged**